packages/docusaurus/bin/docusaurus.mjs

```diff
@@ -104,6 +104,24 @@ cli
   )
   .action(deploy);
 
+/**
+ *
```

Comment:
Also remove this line

Reply:
I was hoping linting would catch some of these but I was wrong, thanks for the review.

Reply:
One day we will have a JSDoc linter, but not at the moment. Prettier cannot analyze much about JSDoc content, so this has to be manual.